packages/expo/src/Linking/Linking.ts

```diff
@@ -189,9 +189,4 @@ newLinking.makeUrl = makeUrl;
 newLinking.parse = parse;
 newLinking.parseInitialURLAsync = parseInitialURLAsync;
 
-// TODO: remove this on SDK 38 after adding to EXLinking
-if (Platform.OS === 'ios') {
-  newLinking.openSettings = () => newLinking.openURL('app-settings:');
```

Comment:
Why would we add more native code when it already works in JS?

Reply:
better to not depend on hardcoded `'app-settings:'` url and instead use `UIApplicationOpenSettingsURLString`

Reply:
I could see `app-settings:` being used as a URI scheme though, like `app-settings://camera` which could scale to work with Android better.

Reply:
people should use intentlauncher on android for this - https://docs.expo.io/versions/v37.0.0/sdk/intent-launcher/

we could potentially make a higher level api around this but for now the idea is to just make sure we provide the implementation for `Linking.openSettings()` that matches upstream